### ISSUE TITLE
"Release Date" and "Genre" fields are required but visual information is not there which show this fields are required: A11y_ASP.NET_Movies_Create New_Label or Instructions

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/aspnetcore#56312]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
## GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #BM_ASP.NET_Web_Jun2024; #.NETCore; #WCAG3.3.2; #DesktopWeb; #Win11; #FTP; #ChromiumEdge; #Label or Instructions; #Regressed:09-20-24; #Repro;
## Environment Details:
App name: ASP.NET
Window Version: Win 11
OS Build: 22631.3737
Chromium Edge Version 126.0.2592.61 (Official build) (64-bit)
## Repro Steps:
1. Download the "RazorPagesApp" zip file and run .exe file.
2. Command prompt will open, TAB to "http://localhost:5000" link and press CTRL + ENTER key.
3. TAB to "Movies" tab and hit ENTER key. TAB to "Create New" button and hit ENTER key.
4. TAB to "Release Date" and "Genre" fields are required but visual information is not there which show this fields are required.
## Actual Result:
"Release Date" and "Genre" fields are required but visual information is not there which show this fields are required.
## Expected Result:
"Release Date" and "Genre" fields are required so visual information should be there which show this are fields are required. For example, there should be required text in parentheses brackets beside label or else asterisk mark (*) should be provided and at top text should be there like asterisk mark are required fields.  
## User Impact:
User will not know "Release Date" and "Genre" fields are required.
## Attachment
https://github.com/dotnet/aspnetcore/assets/124254981/8a9b37b3-f198-4288-9ea8-3c2e67a868d9

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by kapilvaishna** [comment]
issue is still repro 

![image](https://github.com/user-attachments/assets/4b263568-2cfa-452c-9c7e-c3687e27db8d)

**Comment by PoojaNamde** [comment]
#Regressed:11-12-24;

@mkArtakMSFT 

Verified the issue in latest environment. Issue is partially fixed as there is not asterisk provided for "Price" field. Please find below screenshot for reference.


![Image](https://github.com/user-attachments/assets/87449b4a-9c20-48a5-8f28-2c713e0ac43e)

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:12-06-24;

Verified the issue in latest env, issue is fixed hence closing the bug.
https://github.com/user-attachments/assets/6421649a-011a-44ee-98cd-446d369d46f9
